### DISCUSSION TITLE
is magit alone in the universe?

### DISCUSSION THREAD

**Discussion by amtoine** [magit/magit#4778] [Support] [answered]
hello there :wave: :yum: 

## my situation
i've been enjoying `magit` a TON when i used to use _Emacs_ :star_struck: 
so powerfull
so close to the CLI
so great
really :eyes:

however now, i'm not using _Emacs_ anymore and i kinda miss _Magit_ but do not want to use _Emacs_ just for it...

## what i'm looking for
**are there any know "forks" of `magit/magit` written for the terminal and not as an _Emacs_ plugin?** :thinking: 

if yes, i'd love to know where they are
if not, well... i'll have to write my own :wink: :muscle: 

## current _state of the art_ known to me
i've seen:
- [kahole/edamagit](https://github.com/kahole/edamagit) for `vscode` written in `typescript`
- [ananthakumaran/monky](https://github.com/ananthakumaran/monky) for `hg` written in _Emacs_ `lisp`
- [TimUntersberger/neogit](https://github.com/TimUntersberger/neogit) for `neovim` written in `lua`

but none of them really are for `git` AND the terminal
and i'm not really interested in the languages nor the editors :confused:

**Comment by tarsius** [comment, marked as answer]
Not alone, but I like to think that what NealStephenson [said about Emacs](https://www.emacswiki.org/emacs/NealStephenson), can also be said about Magit. :sun_with_face: 

That being said, I just turned my bookmark list into the wiki page titled [other tools](https://github.com/magit/magit/wiki/Other-Tools).

**Reply by amtoine** [reply]
> Not alone, but I like to think that what NealStephenson [said about Emacs](https://www.emacswiki.org/emacs/NealStephenson), can also be said about Magit. sun_with_face
> 
ooh wow :open_mouth:
the analogy is quite nice :yum: 

> That being said, I just turned my bookmark list into the wiki page titled [other tools](https://github.com/magit/magit/wiki/Other-Tools).
>
oooowie :star_struck: 
that looks amazing, i'll have a look thanks :relieved:

**Reply by tarsius** [reply]
You're welcome!
